### PR TITLE
Merge facebook master

### PR DESCRIPTION
**Pull request by wmonk**

### CODE REVIEW

File: packages/react-scripts/template/README.md [hunk 1]
```diff
@@ -174,13 +178,54 @@ Some editors, including Sublime Text, Atom, and Visual Studio Code, provide plug
 
 They are not required for linting. You should see the linter output right in your terminal as well as the browser console. However, if you prefer the lint results to appear right in your editor, there are some extra steps you can do.
 
+<<<<<<< HEAD
```
Comment (wmonk) [inline packages/react-scripts/template/README.md]: Need to fix this

File: packages/react-scripts/template/README.md [hunk 2]
```diff
@@ -1034,6 +1091,17 @@ To override this, specify the `homepage` in your `package.json`, for example:
 
 This will let Create React App correctly infer the root path to use in the generated HTML file.
 
+#### Serving the Same Build from Different Paths
+
+>Note: this feature is available with `react-scripts@0.9.0` and higher.
```
Comment (wmonk) [inline packages/react-scripts/template/README.md]: Update this

**Comment by bang88** [comment]
Could you merge this PR and publish it to npm,I will start a new project based on your scripts,Thanks.

**Comment by wmonk** [comment]
@bang88 released in 1.1.6

**Comment by martinnov92** [comment]
### Thank you!!!

**Pull request merged**